README.rst

```diff
@@ -2,24 +2,19 @@
 Aqueduct
 ========
 
-Framework for performance-efficient prediction.
+Framework for running ML model predictions in the most efficient way.
```

Comment:
This sentence is the subject of many of our discussions. We are not yet ready to change it to another

Reply:
All other changes in this PR is good!